.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,139 @@
+                      popd
+
+                      pushd /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google
+                      find . -type f -not -wholename "./.git*" -not -wholename "./vendor*" -not -name ".travis.yml" -not -name ".golangci.yml" -not -name "CHANGELOG.md" -not -name "GNUmakefile" -not -name "docscheck.sh" -not -name "LICENSE" -not -name "README.md" -not -wholename "./examples*" -not -name "go.mod" -not -name "go.sum" -not -name "staticcheck.conf" -not -name ".go-version" -not -name ".hashibot.hcl" -not -name "tools.go"  -exec git rm {} \; > /dev/null
+                      popd
+
+                      bundle exec compiler -a -e terraform -o $GOPATH/src/github.com/terraform-providers/terraform-provider-google/ > /dev/null
+                      bundle exec compiler -a -e terraform -o $GOPATH/src/github.com/terraform-providers/terraform-provider-google-beta/ > /dev/null
+                      bundle exec compiler -a -e inspec -o /home/runner/work/magic-modules/inspec-gcp > /dev/null
+                      bundle exec compiler -a -e ansible -o /home/runner/work/magic-modules/ansible_collections_google > /dev/null
+                      bundle exec compiler -a -e terraform -f validator -o $GOPATH/src/github.com/GoogleCloudPlatform/terraform-google-conversion > /dev/null
+                      COMMIT_MSG=$(git log -1 --pretty=%B $sha)
+
+                      pushd /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google-beta
+                      git add . && git commit -m "$COMMIT_MSG" || true
+                      popd
+
+                      pushd /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google
+                      git add . && git commit -m "$COMMIT_MSG" || true
+                      popd
+
+                      pushd /home/runner/work/magic-modules/inspec-gcp
+                      git add . && git commit -m "$COMMIT_MSG" || true
+                      popd
+
+                      pushd /home/runner/work/magic-modules/ansible_collections_google
+                      git add . && git commit -m "$COMMIT_MSG" || true
+                      popd
+
+                      pushd /home/runner/work/magic-modules/go/src/github.com/GoogleCloudPlatform/terraform-google-conversion
+                      git add . && git commit -m "$COMMIT_MSG" || true
+                      popd
+                  done
+                  git branch -f downstream-master
+
+            - name: Update downstream-master branch in this repo.
+              uses: ad-m/github-push-action@master
```

Comment:
We shouldn't be passing auth creds to a GitHub action controlled by someone outside the org. The action looks pretty trivial; it should be easy to just inline the `git push` call .